.github/workflows/build-sphinx.yml

```diff
@@ -31,46 +18,22 @@ jobs:
             - name: Set up python
               uses: actions/setup-python@v1
               with:
-                  python-version: 3.8
+                  python-version: "3.10"
 
             - name: Setup conda environment
               uses: conda-incubator/setup-miniconda@v2
               with:
                   activate-environment: env
                   auto-update-conda: true
-                  python-version: 3.8
+                  python-version: "3.10"
                   channels: conda-forge
 
             - name: Install python dependencies with conda
               run: conda env update --file environment.yml
 
-            - name: Check for unused files

```

Comment:
I added this step after removing a bunch of unused files in this repo. Since we rarely add files I no longer think it's useful. And it would have to be updated slightly to work on ubuntu.